CIP-066/README.md

```diff
@@ -0,0 +1,426 @@
+          "WHHoY9F3zeDMaxUoXTcUL9iA7ymZRKdmxsY5sRoPQUjwqnNad2kB7z9u2kZdpPzt",
+          "mKPRPaEvdFjCpTscFimjMNH34h1V7rgFXLBNunV5VXtYrFNKnWVaNwY1ZPcrpqcQ",
+          "g9c3Qi21mNMWPc5XLANX6Rja7ALPp9qy65KpVR1p8eWvEdDkf9uoJQfNMgZxKvVm",
+          "wP3s3Q5FGyi8WisgPoncLQG9RS5EgrG6C7m25fTiV11fEBQU5Aiec5xj3tf7dj8u",
+          "VpQpKmSF7Ayv915VtEZCCJy2cLYMQVD2CfwRcimcaeQommhmNzyo5NnsztS2SMa4",
+          "JaSqV9pr2fbKAkRPCVrpGYa2A1y7apXZ9vJEzLRjhdgbCEAus7gJ8XZnQHD4haXh",
+          "TMHYCcCaYc2s5CGPz3MsP713MgStMXiYNf6PGNJ62pgmqKCu8FMuiKqo5Xj3AEKX",
+          "U5cNivJSU6eJ6u3HFrdWougyfAmodfYx8CHfWCMkcz1UZgNf7q45fXgRw7v6TGwL",
+          "DXgGbcuCPcgmkHw6X7fzDaViAuPnJdZMtbmcDbursH3uUoFdxhfWPeTVzSXgwUB6",
+          "ry2zgqfK4TssLAHghqGLq3dtqV2t1kZxuTnt81hQkN3Wdo4MpkBExC12RV89RJwe",
+          "tqodhMipD9Y8WUYEw3Uu2WGy4U1cu1Qt5c9rc6vPrJV6YR2hJ5tRpaCmWFLxppXz",
+          "3j4yjLxPUjRVtBoJoekB1NJG4o84TzcVohpi5naLABtCp1K8tiNXvRdCafXFcZBh",
+          "VmFJS1mp4vnyDqHsLzrbbQWsdD8SSNoXoArMrihZj4xwneUC2mE2BFeGic58kx84"
+        ]
+      ],
+      "publicKey": [
+        [
+          "MIICIjANBgkqhkiG9w0BAQEFAAOCAg8AMIICCgKCAgEAowyU5YxcEYGUdJO/7XZw",
+          "aymotFny7qocYQPRnFDUDXwLwLoMFpiTeUKBX79dULeTPMHma2m3JByzqeSMfEaL",
+          "bxeIuIctyG9ZiCnSEx2mGEpKIsHwgGcqPNfdfa0gwXZerWtH5+6YgSuLW49L5Byx",
+          "Tn9fF77gnOqrj4xRriCAneQrZdagi+rk6QoLPEYbM7Ctvw4x59scYf3Lo0Ak495T",
+          "IbjjyFo6mq+bMJWLAmSsaeSX5H3kuZTOcv5h4VloRlPiScuuVLotP3cH2afvM8wp",
+          "fsRe1TOgZLO7azCv6xb7leIUmBYDGDgctHvxmN/bUvw1iflOtHkMH/EeXwPQ7BBQ",
+          "3+NHPLhWvygrOXeBNhJKq1uZCyJQRKcVP6z0r/McEvjRiJOAffvD0YN10n+C4eAP",
+          "byW9h+fu3e2D4gnjLVlXFCdVayEJt0nK0PTMzqfTeYmcDfJn+yj9nQzHytq4WqIr",
+          "vo7wn/goKUMiqQwy9YVgyu89EgswaDFqq9r/LTrjca7O84Jf7s2kCv2qpsuuCF+4",
+          "IqI1r3b0HxisKVNKKcl81rFgL9Uf9pkqUg4k7Tty+iiI7kfv3yld9ESLO5bSlHcU",
+          "rJzBm2agIyPUdnXnUg+CbBONFgY7ayWYnmAcCBBs/dTFrcUx7guhl4IPXY/8MMjS",
+          "rZqb/5Uml4KdVHHUDaBnkQ8CAwEAAQ=="
+        ]
+      ]
+    }
+  ]
+}
+```
+
+#### 2. Collection Token
+
+```
+{
+  "725":{
+    "version": "1.0",
+    "fdb275fb75db33313626e0361fc763b6995e97db312be87590e48aa1": {
+        "CIP0066Test": {
+            "type": "Ed25519VerificationKey2020",
+            "files": [
+                {
+                    "src": "ipfs://QmacjTQe5q1ur2FbqW9TehScoP7cZEcAWnSA6gz4FEEMi2",
+                    "name": "CIP-0066_NMKR_IAMX",
+                    "mediaType": "application/ld+json"
+                }
+            ],
+            "@context": "https://github.com/IAMXID/did-method-iamx"
+        }
+    }
+  } 
+}
+```
+
+## Cross-Chain Compatibility
+
+To make this standard cross-chain compatible it is possible to set the value of the src parameter under files to a unique identifier on another blockchain, where the DID might be located.
+
+## Backwards Compatibility
+
+To keep NFT metadata compatible with changes coming up in the future, we use the **`version`** property.
+
+## Rationale
```

Comment:
Some unanswered questions in the document that would deserve an explanation in this section:

- Why bother with two approaches (collection token vs identity token) instead of a single one. It seems that one is clearly a superset of the other, and thus, make the other redundant. If the claim is about offering a simple alternative for those who don't want to go the "complex" way then I'd argue that this is misleading at most as, most implementations will likely end up implementing both making the system ultimately more complex than if there was only one generic and capable way. 

- How does exactly are the metadata tied to an NFT collection (beyond the policy id)? Can anyone publish identity metadata at any time after the mint? Does this has to happen during the mint? If so, then, what's the justification w.r.t the current metadata spoofing issues affecting CIP-0025. 

- Why adopting this particular format? It seems like this is mainly backed by the W3C Recommendation but this is only vaguely mentioned in the specification. If this W3C recommendation is meant to become an industry-grade standard then it'd be nice to highlight this in the rationale section.